src/main/resources/config/WarpDrive.xsd

```diff
@@ -107,6 +107,7 @@
 		<xs:attribute name="blockState" type="xs:string" use="required" />
 		<xs:attribute name="minQuantity" type="xs:string" use="optional" />
 		<xs:attribute name="maxQuantity" type="xs:string" use="optional" />
+		<xs:attribute name="maxTries" type="xs:string" use="optional" />
```

Comment:
Actual parameter name is `maxRetries` while configuration attribute is `maxTries`. Consider keeping the same name all over? also, it's implemented as a maximum of attemps, not a maximum of retries, so `maxAttemps` might be a more appropriate name here.